lms/djangoapps/certificates/views/webview.py

```diff
@@ -418,7 +419,7 @@ def render_html_view(request, user_id, course_id):
     context['share_url'] = share_url
     twitter_url = 'https://twitter.com/intent/tweet?text={twitter_share_text}&url={share_url}'.format(
         twitter_share_text=context['twitter_share_text'],
-        share_url=urllib.quote_plus(share_url)
+        share_url=urllib.quote_plus(smart_str(share_url))
```

Comment:
@ziafazal just for my info, what will happened if twitter_share_text has special characters in it ?